### ISSUE TITLE
White screen of death if you try to post without saving your profile

### ISSUE THREAD

**Issue by kalenjordan** [kalenjordan/magehero#101]
https://twitter.com/jayelkaake/status/558771292342472704

**Comment by navarr** [comment]
Yeah this is ridiculous.  Doesn't even explain I need to save my profile or anything.  Just constantly crashes.

**Comment by kalenjordan** [comment]
The issue is a little different now as of some recent changes.  Now if you try to post without saving your profile, it does actually save the post, but it doesn't link to a user record, which causes some issues.

**Comment by kalenjordan** [comment]
This should be resolveable by going to your profile, saving it, and then saving the post.
